interface/src/app/gene/genes.js

```diff
@@ -0,0 +1,376 @@
+angular.module('adage.gene.search', [
+  'adage.gene.resource'
+])
+
+.config(function($stateProvider) {
+  $stateProvider
+    .state( 'gene_search', {
+      url: '/gene_search',
+      views: {
+        "main": {
+          templateUrl: 'gene/gene-network.tpl.html',
+          controller: ['$scope', 'UserFactory',
+            function($scope, UserFactory) {
+              UserFactory.getPromise().$promise.then( function() {
+                $scope.userObj = UserFactory.getUser();
+              });
```

Comment:
I think in this case it is not necessary because the UserFactory will return 'null' as the user object if no user is logged-in or something goes wrong with the request - so worst thing that can happen at this point is that $scope.userObj is set to null. What do you think @dongbohu ?